samples/interstitialSample/src/main/java/com/inmobi/interstitial/sample/InterstitialAdsActivity.java

```diff
@@ -74,75 +87,81 @@ private void adjustButtonVisibility() {
     }
 
     private void setupInterstitial() {
-        mInterstitialAd = new InMobiInterstitial(InterstitialAdsActivity.this, PlacementId.YOUR_PLACEMENT_ID,
-                new InterstitialAdEventListener() {
-                    @Override
-                    public void onAdLoadSucceeded(InMobiInterstitial inMobiInterstitial) {
-                        super.onAdLoadSucceeded(inMobiInterstitial);
-                        Log.d(TAG, "onAdLoadSuccessful");
-                        if (inMobiInterstitial.isReady()) {
-                            if (mShowAdButton != null) {
-                                mShowAdButton.setVisibility(View.VISIBLE);
+        try {
```

Comment:
`try/catch` block is not necessary for `SdkNotInitializedException` as the intention is to throw exception only for wrong integration. Publishers following the guidelines need not even handle this exception as it will never occur.